src/__tests__/fireEvent.test.js

```diff
@@ -62,9 +62,7 @@ describe('fireEvent', () => {
   });
 
   test('should throw an Error when event handler was not found', () => {
-    const { getByText } = render(
-      <WithoutEventComponent onPress={() => 'this is not passed to children'} />
```

Comment:
please bring this back

Reply:
I was writing the massage for you but you were much faster 😄 

I cannot. The test in that form is wrong or the code has a bug. I believe this cannot be fixed in the code so I've changed the test. The fire event is looking for onPress prop and it was finding that.

After adding wrapper (View in the render method) the test was failing. Previously it worked, because there is a weird "if" [here](https://github.com/callstack/react-native-testing-library/blob/85bccf2f6a6e36d5bf5ef73dcb3c1e6552f84649/src/fireEvent.js#L31) which handles this case.

This test is also failing when wrapping it directly in test, e.g:
```
render(
<View>
      <WithoutEventComponent onPress={() => 'this is not passed to children'} />
</View>
)
```

Previously the test was suppose to check that we throw an error when event handler was not found. But looking at the test case it was there. Do you agree?

Reply:
Seems like it wouldn't really work with `wrapper` option as well. We'd need to check for the wrapper to make it work:
```diff
diff --git a/src/fireEvent.js b/src/fireEvent.js
index 06fa0d3..859b118 100644
--- a/src/fireEvent.js
+++ b/src/fireEvent.js
@@ -27,7 +27,12 @@ const findEventHandler = (
   if (handler && isEventEnabled) return handler;

   // Do not bubble event to the root element
-  if (element.parent === null || element.parent.parent === null) {
+  if (
+    element.parent === null ||
+    element.parent.parent === null ||
+    element.parent.props.nativeID === 'rntl_wrapper' ||
+    element.parent.parent.props.nativeID === 'rntl_wrapper'
+  ) {
     if (hasHandler) {
       return null;
     } else {
diff --git a/src/render.js b/src/render.js
index c304e21..eb499c1 100644
--- a/src/render.js
+++ b/src/render.js
@@ -28,7 +28,11 @@ export default function render<T>(
   { wrapper: Wrapper, createNodeMock }: Options = {}
 ) {
   const wrap = (innerElement: React.Element<any>) =>
-    Wrapper ? <Wrapper>{innerElement}</Wrapper> : <View>{innerElement}</View>;
+    Wrapper ? (
+      <Wrapper>{innerElement}</Wrapper>
+    ) : (
+      <View nativeID="rntl_wrapper">{innerElement}</View>
+    );
```

One thing I didn't realize would be breaking snapshots when adding this wrapper.

Reply:
Maybe we don't need to throw this error at all? Seems like dom testing library doesn't do it https://github.com/testing-library/dom-testing-library/blob/master/src/events.js

Reply:
Maybe not. But still this [if](https://github.com/callstack/react-native-testing-library/blob/85bccf2f6a6e36d5bf5ef73dcb3c1e6552f84649/src/fireEvent.js#L31) will not work, when somebody adds theirs own wrapper.

Reply:
Yea, I'm aware. Nobody complained so far

Reply:
Removed the wrapper for now, we can add it separately along with `baseElement`